### ISSUE TITLE
Feature Request: AIDPC calculator 

### ISSUE THREAD

**Issue by fadero** [micro-manager/micro-manager#1947]
@nicost I'd like to ask for a feature to be added to the On-The-Fly Processor Pipeline: asymmetric illumination differential phase contrast (AIDPC) calculation. This idea was originally conceived by Mehta and Sheppard, 2009 (https://doi.org/10.1364/OL.34.001924). 

Essentially, the input would be two transmitted light images, Right Image (I_R) and Left Image (I_L). The Differential Phase Gradient Image (I_DPC) is equal to:

2* (I_R - I_L) / (I_R + I_L)

This formula (equation 1, Mehta and Sheppard) produces pixel values that always range between -2 and +2, by definition. To keep it simple, it would be useful to cast these values to a 16 bit unsigned integer (for .tifs). 

Additionally, it would be useful to get the average of the two images:

I_AVG = (I_R + I_L) / 2

Which effectively creates a brightfield image. This image is easier to keep in 16 bit, as it's essentially a mean intensity projection between two images.

TL;DR: I'd like an AIDPC image processor that takes in two images (I_R and I_L) and outputs I_DPC and I_AVG.
